beeswax/logs/ad_log.proto

```diff
@@ -105,9 +105,9 @@ message ClickLogMessage {
 // ActivityLogMessage is created for video and activity events
 // like video "START", "SKIP", etc.
 message ActivityLogMessage {
+  // NOTE: auctionid is an opaque unique id. Client cannot rely on the meaning
```

Comment:
Is this actually true for type `base.EventId`? An `EventId` has fields in it that have real meaning and can be used. They're encoded in proto and we are vowing to not break them as a result. The values that shouldn't be relied on are those associated with the `*_str` fields, like `auctionid_str`. 

E.g. when we updated `EventId` to include `imp_idx` it was done in protobuf as a backwards compatible change, but the associated `auctionid_str` would have caused code that was parsing it to break. 

Anyways, long winded way of saying that I think these comments should be specific to `auctionid_str` and not `auctionid`.